### PR TITLE
feat(rms/policy): add new policy assignment sdk

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
Add new sdk support for RMS service.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
  will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: openstack/rms/v1/policyassignments/requests.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+// Delete is a method to remove the specified policy assignment using its ID.
+func Delete(c *golangsdk.ServiceClient, domainId, assignmentId string) error {
+	_, err := c.Delete(resourceURL(c, domainId, assignmentId), &golangsdk.RequestOpts{
+		MoreHeaders: requestOpts.MoreHeaders,
+	})
+	return err
+}
+
+// List is a method to query the list of the policy defunitions using given parameters.
```
Comment (ShiChangkuo) [inline openstack/rms/v1/policyassignments/requests.go:151]:
```suggestion
// List is a method to query the list of the buildin policy definitions using given parameters.
```
defunitions or definitions ?

File: openstack/rms/v1/policyassignments/requests.go [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+func List(client *golangsdk.ServiceClient) ([]PolicyDefinition, error) {
```
Comment (ShiChangkuo) [inline openstack/rms/v1/policyassignments/requests.go:152]: the function name should be **ListDefinitions**

File: openstack/rms/v1/policyassignments/results.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+type PolicyParameterDefinition struct {
+	// The name of the parameter definition.
+	Name string `json:"name"`
+	// The description of the parameter definition.
+	Description string `json:"description"`
+	// The allow value list of the parameter definition.
+	AllowedValues []interface{} `json:"allowed_values"`
+	// The default value of the parameter definition.m
+	DefaultValue string `json:"default_value"`
+	//
```
Comment (ShiChangkuo) [inline openstack/rms/v1/policyassignments/results.go:72]: the comments is missing

File: openstack/rms/v1/policyassignments/requests.go (hunk 1 above)
Comment (Lance52259) [reply openstack/rms/v1/policyassignments/requests.go]: done.
built-in policy definitions

File: openstack/rms/v1/policyassignments/requests.go (hunk 2 above)
Comment (Lance52259) [reply openstack/rms/v1/policyassignments/requests.go]: done.

File: openstack/rms/v1/policyassignments/results.go (hunk 3 above)
Comment (Lance52259) [reply openstack/rms/v1/policyassignments/results.go]: fixed.

**Pull request merged**